libs/blocks/global-navigation/utilities/keyboard/mobilePopup.js

```diff
@@ -6,17 +6,20 @@ import {
   getOpenPopup,
   selectors,
 } from './utils.js';
-import { closeAllDropdowns, logErrorFor } from '../utilities.js';
+import { closeAllDropdowns, logErrorFor, setActiveDropdown } from '../utilities.js';
 
 const closeHeadlines = () => {
   const open = [...document.querySelectorAll(`${selectors.headline}[aria-expanded="true"]`)];
   open.forEach((el) => el.setAttribute('aria-expanded', 'false'));
+  // Shift active class back to the parent of the first closed headline
+  setActiveDropdown(open[0]);
```

Comment:
After the merge of https://github.com/adobecom/milo/pull/823, it would be good to double-check that this is still working as expected. Same for all other newly added `setActiveDropdown` calls from this PR.

Reply:
Actually only integrated `setActiveDropdown` after rebasing to #823  yesterday